benchmark/charset_inclusion.rb

```diff
@@ -4,9 +4,11 @@
 # only characters from the KSUID Base62 charset.
 
 require 'benchmark/ips'
+require 'set'
 
 EXAMPLE = '15Ew2nYeRDscBipuJicYjl970D1'
 CHARSET = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
+CHARSET_SET = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'.split.to_set
```

Comment:
**issue:** This isn't splitting how you expect. Instead, you end up with a set of `Set.new(['0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'])`, which is quite different and doesn't work! 😅 

```suggestion
CHARSET_SET = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'.split('').to_set
```

With the correct version, `Regexp#match?` is the fastest on all Rubies by my benchmarks.

Reply:
Oops, that's embarrassing 😅 You're totally right. I'll add some specs to `Base62.compatible?`. I have a few more ideas how to improve the performance here, so maybe this work won't be totally in vain though

Reply:
Don't be too embarrassed; I thought it was doing the right thing too! At least we can be embarrassed together.